config/stl_hash.m4

```diff
@@ -1,4 +1,4 @@
-# Copied from the protobuf package, http://code.google.com/p/protobuf/
+	# Copied from the protobuf package, http://code.google.com/p/protobuf/
```

Comment:
I don't even know where those spaces came from... I will update the url but I updated the macro's myself